lib/l10n.js

```diff
@@ -13,456 +13,569 @@ var l10n = {
+        es: "Fight for the Future (FFTF), su Fondo de Educación (FFTFEF), y {{partner}} le enviará información de votante y otras alertas de acción.",
+        ko: "이 앱의 주관 단체인 FFTF 및 FFTFEF (\“미래를 쟁취하자]\”) 에서 선거 정보를 포함한 여러가지 정보를 보내드릴 것입니다. "
     },
     msg_sms_fftf_stop: {
         en: "(To stop messages from FFTF & FFTFEF, text STOP.)",
-        es: "(Para detener mensajes de FFTF & FFTFEF, envíe STOP.)"
+        es: "(Para detener mensajes de FFTF & FFTFEF, envíe STOP.)",
+        ko: " 이 앱의 주관 단체인 FFTF 및 FFTFEF (\“미래를 쟁취하자]\”) 에서 선거 정보를 포함한 여러가지 정보를 보내드릴 것입니다. "
     },
     msg_share_facebook_messenger: {
         en: "Now, there's one last important thing. Please pass on the <3 and register some friends! Tap the button below to share me on Facebook.",
-        es: "Ahora hay una última cosa importante. Por favor, difunde el <3 y registre algunos amigos! Haga click en el botón abajo para compartir en Facebook"
+        es: "Ahora hay una última cosa importante. Por favor, difunde el <3 y registre algunos amigos! Haga click en el botón abajo para compartir en Facebook",
+        ko: "더 많은 사람들이 유권자 등록을 할 수 있도록 널리 알려주세요! 아래의 버튼을 눌러서 페이스북에서 공유 해 주세요."
     },
     prompt_restart: {
         en: "This will restart your HelloVote registration! Reply OK to continue or BACK to go back.",
-        es: "Esto va a empezar de nuevo tu inscripción HolaVote! Responde SI para continuar o REGRESA para regresar"
+        es: "Esto va a empezar de nuevo tu inscripción HolaVote! Responde SI para continuar o REGRESA para regresar",
+        ko: "유권자 등록을 처음부터 다시 시작하시겠어요? 처음부터 시작하려면 ok, 이전 단계로 돌아가려면 back 을 입력 해 주세요."
     },
     prompt_us_citizen: {
         en: "Are you a US citizen? (yes/no)",
-        es: "¿Eres un ciudadano de los Estados Unidos? (si/no)"
+        es: "¿Eres un ciudadano de los Estados Unidos? (si/no)",
+        ko: "미국 시민권자입니까? (Yes/no)"
     },
     prompt_legal_resident: {
         en: "Are you a current legal resident of {{settings.state}}? (yes/no)",
-        es: "¿Eres un residente legal de {{settings.state}}? (si/no)"
+        es: "¿Eres un residente legal de {{settings.state}}? (si/no)",
+        ko: "현재 {{settings.state}} 의 법적 주민입니까? (yes/no)"
     },
     prompt_military_or_overseas: {
         en: "Are you a military or overseas voter? (military/overseas/no)",
-        es: "“¿Eres un votante militar o del el extranjero? (militar/extranjero/no)"
+        es: "“¿Eres un votante militar o del el extranjero? (militar/extranjero/no)",
+        ko: "군인 신분으로서 투표, 또는 미국 외 해외에서 투표를 하고자 하십니까? 군인이면 military, 해외 투표는 overseas, 둘 다 아니면 no 를 입력 해 주세요."
     },
     error_military_or_overseas: {
         en: "Sorry, I didn't get that. Please answer (military/overseas/no)",
-        es: "Lo siento, no entendí eso. Por favor contesta (militar/extranjero/no)"
+        es: "Lo siento, no entendí eso. Por favor contesta (militar/extranjero/no)"ES: "Lo siento, no entendí eso. Por favor conteste (militar/extranjero/no)"
```

Comment:
this looks malformed, please ensure there is only one key for each language

Reply:
will reformat